src/Kunstmaan/NodeSearchBundle/EventListener/NodeIndexUpdateEventListener.php

```diff
@@ -18,17 +20,23 @@ class NodeIndexUpdateEventListener implements NodeIndexUpdateEventListenerInterf
     /** @var ContainerInterface */
     private $container;
 
+    /** @var EntityManagerInterface */
+    private $em;
+
     /** @var NodePagesConfiguration */
     private $nodePagesConfiguration;
 
     /** @var array */
     private $entityChangeSet;
 
-    /**
-     * @param ContainerInterface $container
-     */
-    public function __construct(/* NodePagesConfiguration */ $nodePagesConfiguration)
+    public function __construct(/** NodePagesConfiguration */
+        $nodePagesConfiguration, /** EntityManagerInterface */
+        $em = null)
     {
+        if (null !== $em) {
+            $this->em = $em;
+        }
+
```

Comment:
Also add a deprecation notice if the entity manager is not passed, so users know this argument will be required in 6.0